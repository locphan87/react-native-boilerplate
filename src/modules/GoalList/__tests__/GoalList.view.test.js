import React from 'react'
import { shallow } from 'enzyme'

import { snapTest } from '../../../utils/test.util'
import GoalListView from '../GoalList.view'

const props = {
  goalList: [],
  OPEN_DETAILS: jest.fn()
}
const wrapper = shallow(<GoalListView {...props} />)

describe('GoalList View', () => {
  snapTest(wrapper, [
    {
      props,
      description: 'basic render'
    }
  ])
})
