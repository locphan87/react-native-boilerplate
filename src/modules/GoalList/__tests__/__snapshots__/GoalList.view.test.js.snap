// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoalList View snapshots basic render 1`] = `
<StyledWrapper>
  <GoalList
    list={Array []}
    onPress={[MockFunction]}
  />
</StyledWrapper>
`;
